src/article/article-route.js

```diff
@@ -1,10 +1,11 @@
 const { Router } = require("express");
 
-const { findAll, updateArticle, create, deleteAll } = require("./article-controller");
+const { findAll, updateArticle, create, deleteAll,uploadFiles } = require("./article-controller");
 
 const articleRouter =   Router(); 
 articleRouter.get('/list', findAll);
 articleRouter.put('/update',updateArticle);
 articleRouter.post('/create',create);
 articleRouter.delete('/delete',deleteAll);
+articleRouter.post('/upload-files', uploadFiles);
```

Comment:
Change name to singular since the endpoint only supports one file to be uploaded to it